

def error_message(number):
    message = f'Test{number} Fail'
    return message